requirements/edx/github.txt

```diff
@@ -82,7 +82,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.4#egg=XBlock==0.4.4
 -e git+https://github.com/edx/edx-submissions.git@0.1.3#egg=edx-submissions==0.1.3
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
 git+https://github.com/edx/i18n-tools.git@v0.2#egg=i18n-tools==v0.2
-git+https://github.com/edx/edx-oauth2-provider.git@0.5.8#egg=edx-oauth2-provider==0.5.8
+git+https://github.com/edx/edx-oauth2-provider.git@cdyer/make-way-for-oauth-toolkit#egg=edx-oauth2-provider==0.5.9a1
```

Comment:
Also, the version should get a major version bump, given your changes are backward-incompatible.